docs/adr/0004-modular-monolith-bounded-contexts.md

```
@@ -0,0 +1,231 @@
+
+| Context     | Owns                                                                   |
+|-------------|------------------------------------------------------------------------|
+| `detection` | Hot pipeline. Tables: `events`, `processes`, `hosts`, `alerts`, `alert_events`. |
+| `rules`     | Detection content + blocklist policy. Tables: `policies`.              |
+| `response`  | Agent commands. Tables: `commands`.                                    |
+| `endpoint`  | Host enrollment + host-token verification. Tables: `enrollments`.       |
+| `identity`  | Operator users, sessions, login. Tables: `users`, `sessions`.           |
+
+Each context follows this layout:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)